common-arcana.lic

```diff
@@ -133,6 +140,28 @@ module DRCA
     DRC.retreat(settings.ignored_npcs) unless spell['skip_retreat']
   end
 
+  def runestone(spell, settings)
```

Comment:
💡 This method looks like it's the start of the casting from a runestone? Would `prepare_runestone` or `prepare_to_cast_runestone` be a more accurate name for this method?

Reply:
Will review